app/clients/facebook/messenger_client.rb

```diff
@@ -0,0 +1,31 @@
+class Facebook::MessengerClient
+  def send_message(message)
+    response = faraday_connection.post("me/messages?access_token=#{page_access_token}", message)
+    response.body if response&.success?
+  end
+
+  def user_info(sender_id)
+    response = faraday_connection.get("#{sender_id}?access_token=#{page_access_token}")
+    response.body if response&.success?
+  end
+
+  private
+
+  def faraday_connection
+    connection = Faraday.new(url: api_url) do |faraday|
+      faraday.response :json
+      faraday.request :json
+      faraday.request :url_encoded
+      faraday.adapter Faraday.default_adapter
+    end
+    connection
+  end
+
+  def page_access_token
+    ENV['FACEBOOK_PAGE_ACCESS_TOKEN']
```

Comment:
Я бы предложил так
```suggestion
    ENV.fetch('FACEBOOK_PAGE_ACCESS_TOKEN')
```
чтобы быть уверенным, что такая переменная там есть